src/test/groovy/com/github/roroche/plantuml/android/AndroidProjectTypeTest.groovy

```diff
@@ -0,0 +1,44 @@
+package com.github.roroche.plantuml.android
+
+import com.github.roroche.plantuml.urls.MainOutputUrlsFactory
+import com.github.roroche.plantuml.urls.Urls
+import org.gradle.api.Project
+import org.gradle.testfixtures.ProjectBuilder
+import org.junit.jupiter.api.Test
+
+class AndroidProjectTypeTest {
+    private final Project project = ProjectBuilder.builder().build()
+
+    @Test
+    void testProjectIsAndroidApplication() {
+        project.apply plugin: "com.android.application"
+        project.android {
+            compileSdkVersion 21
+        }
+        project.evaluate()
```

Comment:
I think we can reproduce the mechanism found here: https://github.com/RoRoche/plantuml-gradle-plugin/blob/master/src/test/groovy/com/github/roroche/plantuml/BuiltProject.groovy

I suggest a new interface such as:

```kotlin
interface TestableProject {
  fun toProject(): Project
}
```

and then you can put the logic to prepare a project such as:

```kotlin
class TestableAndroidApplication: TestableProject {
  fun toProject(): Project {
    ProjectBuilder.builder().build()
    project.apply plugin: "com.android.application"
    project.android {
      compileSdkVersion 21
    }
    project.evaluate()
    return project
  }
}
```

This will remove duplicate code in test.